### PR TITLE
Adds two new post-threshold Elf-A mutations

### PR DESCRIPTION
**Pull request by Ilysen**
#### Summary
```SUMMARY: Content "Post-threshold Elf-A mutants can mutate new traits to make them closer to nature."```

#### Purpose of change
Despite recent changes, Elf-A still has very little going for it post-threshold, and some unique traits should help to make them a bit more interesting.

#### Describe the solution
Adds a few new mutations:
* Keening Voice, which does nothing.
* Wailing Voice, mutated from Keening Voice, which has a 1-in-3 chance to induce a frenzy in nearby animals when shouting.
* Beast Rapport, mutated from Wailing Voice, which can be enabled and disabled to induce frenzy in nearby animals when shouting. Requires elf-a threshold.
* Rosebuds, mutated from Flowering, which resembles triffid bulbs and emits pheromones that make them ignore you. Requires elf-a or plant threshold.

Frenzied creatures attack everything in sight, including the player and other frenzied animals, and won't flee from combat under any conditions. Points of note:
* When frenzying, an animal has a 1-in-3 chance of being friendly to you for a short time.
* If not friendly, an animal has a 1-in-5 chance of being stunned for 5 turns immediately.
* Frenzy lasts 5 minutes, or 50 turns.
* Even creatures that never fight normally will charge into combat while frenzied. Some are actually great combatants, like deer, due to their speed being balanced around fleeing and not fighting.

#### Describe alternatives you've considered
Making the rapport always turn animals friendly to you and towards one another. I ultimately considered that to be too magic-y, whereas an alien sound that drives animals to madness from mutated vocal chords is still workable within the bounds of Cataclysm.

Granting the ability to tame animals instead of frenzying them. This was the original suggestion this was based off, but I thought the frenzy was more interesting both thematically and mechanically.

#### Additional context
image

### CODE REVIEW

**Comment by DavisBroda** [comment]
Triffids aren't really a part of nature in the same way animals are. They are just as alien as the Mycus, or the Blob. They are both sapient, and have almost nothing in common with earth plants, so I doubt a pheremone trick would work on them. There's a bit of a discussion about how to talk/interact with triffids in a [forum thread](https://discourse.cataclysmdda.org/t/talking-to-plants-interacting-with-the-triffids/19109) which might be worth checking out in relation to this.

**Comment by Ilysen** [comment]
Hmm; after reading through, it does seem that you're right. I can axe the triffid neutrality part of Rosebuds for the time being, and come back to it later once triffids have been more fleshed-out.

File: data/json/mutations.json [hunk 1]
```diff
@@ -5830,6 +5842,38 @@
...
+    "changes_to": [ "KEENING3" ],
+    "social_modifiers": { "persuade": -5, "intimidate": -5 }
+  },
+  {
+    "type": "mutation",
+    "id": "KEENING3",
+    "name": "Beast Rapport",
+    "points": 3,
+    "prereqs": [ "ANIMALEMPATH2", "KEENING2" ],
+    "description": "You've developed control over your mutated vocal chords.  While active, yelling will emit a high-pitched scream that drives even the most docile of animals that can hear you into a killing frenzy.  The animals might infight, and their list of targets often includes you, so using it from a safe distance is wise.",
```
Comment (thquinn) [inline data/json/mutations.json]: I'd get rid of "that can hear you."

File: data/json/mutations.json [hunk 2]
```diff
@@ -2181,8 +2181,20 @@
...
+    "changes_to": [ "ROSEBUDS" ],
     "social_modifiers": { "lie": 10 }
   },
+  {
+    "type": "mutation",
+    "id": "ROSEBUDS",
+    "name": "Rosebuds",
+    "points": 3,
+    "prereqs": [ "FLOWERS" ],
+    "description": "The tpo of your head is blooming with rosebuds.  They're eye-catching, and have a strong fragrance that makes you pleasant to be around.",
```
Comment (thquinn) [inline data/json/mutations.json]:
```suggestion
    "description": "The top of your head is blooming with rosebuds.  They're eye catching, and have a strong fragrance that makes you pleasant to be around.",
```

File: data/json/mutations.json [hunk 3]
```diff
@@ -5830,6 +5842,38 @@
     },
     "purifiable": false
   },
+  {
+    "type": "mutation",
+    "id": "KEENING",
+    "name": "Keening Voice",
+    "points": 0,
+    "description": "Your vocal chords have mutated, and your voice is slightly higher-pitched.  This serves no practical effect.",
```
Comment (thquinn) [inline data/json/mutations.json]:
```suggestion
    "description": "Your vocal chords have mutated, and your voice is slightly higher pitched.  This serves no practical effect.",
```

File: data/json/mutations.json [hunk 4]
```diff
@@ -2181,8 +2181,22 @@
...
   },
+  {
+    "type": "mutation",
+    "id": "ROSEBUDS",
+    "name": "Rosebuds",
+    "points": 3,
+    "visibility": 10,
+    "ugliness": -6,
+    "prereqs": [ "FLOWERS" ],
+    "description": "The top of your head is blooming with rosebuds.  They're eye-catching, and have a strong fragrance that makes you pleasant to be around.",
```
Comment (thquinn) [inline data/json/mutations.json]:
```suggestion
    "description": "The top of your head is blooming with rosebuds.  They're eye catching, and have a strong fragrance that makes you pleasant to be around.",
```

**Comment by thquinn** [comment]
Cool! Maybe you could add some RNG to the effect durations, to make them a little less predictable?

File: data/json/mutations.json (hunk 4 above)
Comment (Ilysen) [reply data/json/mutations.json]: I think this is proper grammar, I'm not sure.

**Comment by Ilysen** [comment]
That doesn't seem too bad of an idea. I'd like to keep it in the same ballpark, since as a post-threshold mutation, it's a major goalpost for a category, and having it be very random would make it difficult to justify taking. 

It's already pseudo-random, since even after the frenzy ends, the morale and anger persist for a time until they diminish naturally, meaning creatures can change attitudes at different intervals. I think, though, that there's no harm in changing it up a bit, since the vast majority of frenzy's duration comes from that modified anger and morale after the effect itself ends.

File: data/json/mutations.json (hunk 4 above)
Comment (thquinn) [reply data/json/mutations.json]: You usually only need to hyphenate compound adjectives if they come before whatever they're describing: "the rosebuds are eye catching" vs "the eye-catching rosebuds": https://www.quickanddirtytips.com/education/grammar/how-to-use-a-hyphen

To use an example from a mistake I made in a previous PR: "A super concentrated mutagen." Is it a (super) (concentrated) mutagen, a mutagen that's both super and concentrated, or is it a (super concentrated) mutagen? The ambiguity doesn't exist when it comes after: "A mutagen that's super concentrated." You couldn't write two adjectives in sequence that way.

**Comment by thquinn** [comment]
> I'd like to keep it in the same ballpark, since as a post-threshold mutation, it's a major goalpost for a category, and having it be very random would make it difficult to justify taking.

Totally, I feel like +/- 2 turns on a 5-minute effect would even be enough. Just so you can't keep your eye on the clock and go "and the effect ends... now."

> It's already pseudo-random, since even after the frenzy ends, the morale and anger persist for a time until they diminish naturally

Ah, I didn't know that. So the stun is the only effect that's actually predictable?

**Comment by SantosValourion** [comment]
An idea that combines #29407 with this, is a trait after flowering that mutates hair roots, that trait could lead to #29407 but be less effective, take longer etc...

**Comment by thquinn** [comment]
@SantosValourion It certainly makes sense for elves to be able to commune with trees, but the hair roots thing... do they put their heads on the ground and let the roots intermingle with the trees'? It's a strange mental picture for me.

**Comment by SantosValourion** [comment]
@thquinn I think off it as they lay down on there back beside the tree and let there roots dig into the ground and connect with the tree. Idk it is a bit strange, but it seems like an elf thing to do.

File: data/json/mutations.json (hunk 4 above)
Comment (Ilysen) [reply data/json/mutations.json]: Wowie, I thought I was good at English. Thanks, corrected.

**Comment by Ilysen** [comment]
Going on indefinite break - anyone who wants to pick this up can.

**Pull request closed without merging**